bundles/org.openhab.binding.easee/README.md

```diff
@@ -0,0 +1,170 @@
+| siteId                  | yes      | The ID of the site containing the wallbox(es) and circuit(s) that should be integrated into openHAB. The ID of your site can be found out via the sites overview (https://easee.cloud/sites). You just need to click on one of the sites listed there, the id will be part of the URL which is then opened. It will be a number with typically 6 digits. |
+| dataPollingInterval     | no       | interval (seconds) in which live data values are retrieved from the Easee Cloud API. (default = 120)                                                                                                                                                                                                                                                     |
+
+## Thing configuration
+
+It is recommended to use auto discovery which does not require further configuration.
+If manual configuration is preferred you need to specify configuration as below.
+
+### Charger
+
+| Configuration Parameter | Required | Description                                                                                                            |
+|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------|
+| id                      | yes      | The id of the charger that will be represented by this thing.                                                          |
+
+### Mastercharger
+
+| Configuration Parameter | Required | Description                                                                                                            |
+|-------------------------|----------|------------------------------------------------------------------------------------------------------------------------|
+| id                      | yes      | The id of the charger that will be represented by this thing.                                                          |
+| circuitId               | yes      | The id of the circuit that is controlled by this charger.                                                              |
+
+## Channels
+
+The binding only supports a subset of the available endpoints that is provided by the Easee Cloud API.
+The tables below shows all channels that are available and which of them are writable.
+The setting that start with "dynamic" can be changed frequently, the other are written to flash and thus should not be changed too often as this could result in damage of your flash.
+
+### Charger Channels
+
+| Channel Type ID                             | Item Type                | Writable | Description                                          | Allowed Values (write access)                                                                                                                                |
```

Comment:
Header for first column should be "Channel" or similar. It could also be separated into two columns: Channel and Channel Group. Channel Type ID is not of much relevance to users.

Reply:
Channel Type ID is what you need when you use textual configuration. Or maybe it is just the wrong naming of the column? However I can separate it into group and channel.

Reply:
Yes, it's only the naming. I just committed a change to "Channel" in case you wouldn't respond, because I will then fit your PR into next milestone. If you would like to separate into group and channel, please go ahead. I'm leaving in 20 minutes, if it will be quick change, I'll merge your PR by then. :-)

Reply:
Btw, don't worry about the DCO from my commit, I'll fix it when merging.